dataproxy/vercel-edge-functions/run.sh

```diff
@@ -4,4 +4,4 @@ set -eu
 
 yarn install
 
-yarn -s vercel --token=$VERCEL_TOKEN --env VERCEL_DATA_PROXY_URL="$VERCEL_DATA_PROXY_URL" --prod --scope=$VERCEL_ORG_ID --confirm --force 1> deployment-url.txt
+yarn -s vercel --token=$VERCEL_TOKEN --env VERCEL_DATA_PROXY_URL="$VERCEL_DATA_PROXY_URL" --prod --scope=$VERCEL_ORG_ID --yes --force 1> deployment-url.txt
```

Comment:
```suggestion
yarn -s vercel deploy --token=$VERCEL_TOKEN --env VERCEL_DATA_PROXY_URL="$VERCEL_DATA_PROXY_URL" --prod --scope=$VERCEL_ORG_ID --yes --force 1> deployment-url.txt
```

Reply:
It's the same, but I like when it’s explicit too

Reply:
Yep - I needed to double check in the docs if `--yes` also applies to the main command (and their documentation is ignoring the fact pretty much that the main command and `deploy` are the same)

Reply:
Did a commit